### PR TITLE
added doc string to save function

### PR DESCRIPTION
**Pull request by 3bdifatah**
added little description of what the save function does. it's for issue #6 @ewarlock

### CODE REVIEW

**Comment by ewarlock** [comment]
Describes the function well - the only change I would make is for style. I would add a period at the end of the first sentence and capitalize 'If' at the start of the next, since I noticed a lot of the other docstrings are written in a slightly more formal/grammatically strict manner.

**Comment by 3bdifatah** [comment]
good catch. let me know if i can merge and close issue #6

**Comment by ewarlock** [comment]
Looks great, ready to merge!

File: bookstore.py [hunk 1]
```diff
@@ -19,6 +19,9 @@ def __init__(self, title, author, read=False, id=None):
 
 
     def save(self):
+        """ Save function updates book if the ID is in the database.
+        If not it adds the book to the database.
+        """
```
Comment (claraj) [inline bookstore.py:24]: We know we are in a function, so you can simplify this to 
```
""" Update book if the ID is in the database.
        If not, add the book to the database.
"""
```

**Review by claraj — COMMENTED** [review]
Looks good

**Pull request merged**